### DISCUSSION TITLE
Advice about device.

### DISCUSSION THREAD

**Discussion by emilic14** [pcmihnea/HA_WMR500#5] [Q&A] [answered]
Hello,
So I am looking into getting a weather station, preferably compatible with HAS.

And I've liked the visual design of this one, and then I found your work which seems amazing!!
But before moving ahead to buy one and modify it, I was wondering if you could help me out with my questions about the device:

1.- do you recommend it? (as in how does it works with home assistant, and how do the sensors respond, etc) (or would you recommend to look for a different one...) 
2.- love the work you have done, do you think (maybe already exist) the integration to has is possible through rtl433? (one of my goals is to reduce the amount of devices eating energy on my apartment) 
3.- last one, I've seen the device states it has an uv sensor, but then I've also seen that it's an add on attachment that never came out? Can you confirm if it has one or not? 

Thanks!

**Comment by pcmihnea** [comment, marked as answer]
Hi,

Unfortunately, I do not recommend it at all, since it's an old device that's no longer sold for years (2018?) and with zero support from the manufacturer.
The reason I've reverse engineered it is to actually have it as a functional device, since you do need the app (and cloud services, which are down ~2021) to connect it to Wi-Fi and update the firmware, all before you can even get anything out of it remotely.
STandalone (no WiFi), it uses decent sensors chips, but the weather forecast is useless, and compared to other more accurate devices it has a noticeable offset. As a bonus, it also randomly stops reporting the measurements via MQTT, requiring a power cycle :)
The UV and PM10 are each separate modules attached to the outside station, none of which I've ever seen available to buy... 
The communication is bidirectional via 868MHz only, which I've not attempted to reverse engineer, since a SW approach was easier.
If you find it at bargain price, it's nice as a case of hardware platform to build upon your own station - I've found it at ~30$ in 2018 :)).
There's a lot of open source projects, but since I'm limited to where I can actually install it outside, I've relied more on the AccuWeather/OpenWeatherMaps integrations.
Hope it helps!

**Reply by emilic14** [reply]
Thanks a lot! This really helps. 
OK I guess if I find it cheap I'll get it.

Is there one that you would recommend? 

Cheers!

**Reply by pcmihnea** [reply]
I haven't searched through the market for a while, but indoors I'm using thermo-hygro chips from TI - see my other repos. Outdoors, online services.

**Reply by emilic14** [reply]
Are you talking about your HA_HDC1080 repo?
I searched for thermo hygro and It seems I am using similar ones, I use some la crosse mobile assistance ma10018 etc.
Ill have to look more into it.

I am now following your development, impressive work!, thanks!

**Reply by pcmihnea** [reply]
Yes, back when I was searching reliable semiconductor suppliers, that evaluation board had the highest accuracy sensor at a set budget.
Glad to be of help!
